src/titan_db_test.cc

```diff
@@ -1987,7 +1987,7 @@ TEST_F(TitanDBTest, Config) {
 }
 
 #if defined(__linux) && !defined(TRAVIS)
-TEST_F(TitanDBTest, NoSpaceLeft) {
+TEST_F(TitanDBTest, DISABLED_NoSpaceLeft) {
```

Comment:
why disable it

Reply:
keep failing in CI environment. we disabled it previously on travis, too.

Reply:
though it would be great if we can fix it.